exercises/complex-numbers/ComplexNumbersTest.cs

```diff
@@ -0,0 +1,224 @@
+// This file was auto-generated based on version 1.0.0 of the canonical data.
+
+using Xunit;
+
+public class ComplexNumbersTest
+{
+    [Fact]
+    public void Imaginary_unit()
+    {
+        var z1 = new ComplexNumber { Real = 0, Imaginary = 1 };
+        var z2 = new ComplexNumber { Real = 0, Imaginary = 1 };
+        var expected = new ComplexNumber { Real = -1, Imaginary = 0 };
+        Assert.Equal(expected, ComplexNumbers.Mul(z1, z2));
```

Comment:
@ErikSchierboom I'm fine with that approach which will end up with using having pretty much the same solution as they do in the Java track:
https://github.com/exercism/java/blob/master/exercises/complex-numbers/src/example/java/ComplexNumber.java

The `getReal` and `getImaginary` methods are trivially to implement, but that's definitely not the main complexity of the question.

Another thing is once I implement part II, I'll have to convert the integers to be double to represent non-whole numbers with `pi` and `e`.